src/Features/LanguageServer/Protocol/Handler/CodeLens/CodeLensResolveHandler.cs

```diff
@@ -33,12 +33,13 @@ public CodeLensResolveHandler(CodeLensCache codeLensCache)
     public bool RequiresLSPSolution => true;
 
     public LSP.TextDocumentIdentifier GetTextDocumentIdentifier(LSP.CodeLens request)
-        => GetCacheEntry(request).CacheEntry.TextDocumentIdentifier;
+        => GetCodeLensResolveData(request).TextDocument;
```

Comment:
the key line that changed - instead of reading from the cache entry (which in this error scenario doesn't exist) we read from the resolve data (which comes from the request itself).